target/linux/ipq40xx/patches-4.19/182-crypto-qce-fix-ctr-blocksize.patch

```diff
@@ -0,0 +1,11 @@
+--- a/drivers/crypto/qce/ablkcipher.c	2019-10-21 17:18:47.311684603 -0300
++++ b/drivers/crypto/qce/ablkcipher.c	2019-10-21 17:20:32.080765945 -0300
+@@ -292,7 +292,7 @@
+ 		.name		= "ctr(aes)",
+ 		.drv_name	= "ctr-aes-qce",
+ 		.blocksize	= AES_BLOCK_SIZE,
+-		.ivsize		= AES_BLOCK_SIZE,
++		.ivsize		= 1,
```

Comment:
You originally asked me to make `blocksize = 1` not `ivsize = 1` as shown. Did I misunderstand? The patches as shown don't seem to work to get `rfc3686(ctr(aes))` available.

Reply:
You're right, it is the block size that needs to be set to one.  That's embarrassing. 😞
Thanks for pointing it out.